src/utils/contract.ts

```diff
@@ -13,10 +13,10 @@ export function getContractAddressFromNetworkInfo(
   network: OptionalNetwork,
   networkInfo: ContractNetworkInfo,
 ): string | null {
-  if (!network) {
+  if (!network || networkInfo[network]?.address) {
```

Comment:
mm i think this should just be: 
```suggestion
  if (!network || !networkInfo[network].address) {
```
or am i wrong? :thinking: now im confused with the coalescing

Reply:
You are right, I forgot to negate the second part.
But maybe the coalescing should stay. Will try it out.